appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -14,39 +14,39 @@ interface ILinuxRuntimeStack {
 }
 
 export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
-    public async prompt(wizardContext: IAppServiceWizardContext): Promise<IAppServiceWizardContext> {
-        if (!wizardContext.newSiteRuntime) {
-            if (wizardContext.newSiteKind === AppKind.functionapp) {
-                const runtimeItems: IAzureQuickPickItem<string>[] = [
-                    { label: 'JavaScript', data: 'node' },
-                    { label: '.NET', data: 'dotnet' }
-                ];
+    public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
```

Comment:
So why does this one not have the canprompt check?  I was wondering if the extra check was really necessary.  Or maybe an assertion instead (preferably reusing shouldPrompt)